tests/benches/mixnet.rs

```diff
@@ -0,0 +1,51 @@
+use criterion::BenchmarkId;
+use criterion::Criterion;
+use criterion::{criterion_group, criterion_main};
+use futures::StreamExt;
+use mixnet_client::MixnetClientMode;
+use mixnet_client::{MixnetClient, MixnetClientConfig};
+use rand::rngs::OsRng;
+use rand::RngCore;
+
+async fn test_one_message(msg_size: usize) {
+    let (topology, mut destination_stream) = tests::run_nodes_and_destination_client().await;
+
+    let mut msg = vec![0u8; msg_size];
+    rand::thread_rng().fill_bytes(&mut msg);
```

Comment:
Thank you! I wanted to run this part in the 'setup' stage, so that this part is not measured by benchmark. But, I coudn't find a way to send mutual references to the async function run by bench.

Reply:
👍, move it to setup stage

Reply:
Ah, what I suggested was calling the `run_nodes_and_destination_client` at the setup stage because it takes some time to spawn processes. Sorry for confusion. 
I’m not sure if it’s possible.

Reply:
Ah, I suppose `run_nodes_and_destination_client` also needs to be benched. It is possible by using `iter_custom`.

Reply:
Oh, I didn't know about `iter_custom`. From your code, it seems that we can put a `for` loop inside the closure and do some initializations outside of the loop. I'm curious if benchmark measure only the loop. I'll also try to run it. Thank you!